core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairGroup.java

```diff
@@ -52,22 +59,38 @@ public class RepairGroup extends ScheduledTask
 
-    public RepairGroup(int priority, Builder builder)
+    public RepairGroup(final int priority, final Builder builder)
     {
         super(priority);
-        myTableReference = Preconditions.checkNotNull(builder.tableReference, "Table reference must be set");
-        myRepairConfiguration = Preconditions.checkNotNull(builder.repairConfiguration, "Repair configuration must be set");
-        myReplicaRepairGroup = Preconditions.checkNotNull(builder.replicaRepairGroup, "Replica repair group must be set");
-        myJmxProxyFactory = Preconditions.checkNotNull(builder.jmxProxyFactory, "Jmx proxy factory must be set");
-        myTableRepairMetrics = Preconditions.checkNotNull(builder.tableRepairMetrics, "Table repair metrics must be set");
-        myRepairResourceFactory = Preconditions.checkNotNull(builder.repairResourceFactory, "Repair resource factory must be set");
-        myRepairLockFactory = Preconditions.checkNotNull(builder.repairLockFactory, "Repair lock factory must be set");
-        myTokensPerRepair = Preconditions.checkNotNull(builder.tokensPerRepair, "Tokens per repair must be set");
-        myRepairPolicies = new ArrayList<>(Preconditions.checkNotNull(builder.repairPolicies, "Repair policies must be set"));
-        myRepairHistory = Preconditions.checkNotNull(builder.repairHistory, "Repair history must be set");
-        myJobId = Preconditions.checkNotNull(builder.jobId, "Job id must be set");
+        myTableReference = Preconditions
+                .checkNotNull(builder.tableReference, "Table reference must be set");
+        myRepairConfiguration = Preconditions
+                .checkNotNull(builder.repairConfiguration, "Repair configuration must be set");
+        myReplicaRepairGroup = Preconditions
+                .checkNotNull(builder.replicaRepairGroup, "Replica repair group must be set");
+        myJmxProxyFactory = Preconditions
+                .checkNotNull(builder.jmxProxyFactory, "Jmx proxy factory must be set");
+        myTableRepairMetrics = Preconditions
+                .checkNotNull(builder.tableRepairMetrics, "Table repair metrics must be set");
+        myRepairResourceFactory = Preconditions
+                .checkNotNull(builder.repairResourceFactory, "Repair resource factory must be set");
+        myRepairLockFactory = Preconditions
+                .checkNotNull(builder.repairLockFactory, "Repair lock factory must be set");
+        myTokensPerRepair = Preconditions
+                .checkNotNull(builder.tokensPerRepair, "Tokens per repair must be set");
+        myRepairPolicies = new ArrayList<>(Preconditions
+                .checkNotNull(builder.repairPolicies, "Repair policies must be set"));
+        myRepairHistory = Preconditions
+                .checkNotNull(builder.repairHistory, "Repair history must be set");
+        myJobId = Preconditions
+                .checkNotNull(builder.jobId, "Job id must be set");
     }
 
+    /**
+     * Execute repair job.
+     *
```

Comment:
This javadoc needs more info. Something like:
Executes the repair tasks this repairGroup is responsible for. Repair tasks can succeeded or fail, repair tasks blocked by run policy are counted as failed.